code/testsuite/csheets/pf_Cleric.xml

```diff
@@ -2319,28 +2337,6 @@ BIO
 			<virtual>F</virtual>
 			<category>Special Ability</category>
 		</ability_object>
-		<ability_object>
```

Comment:
This change is unexpected. I don't see anything that makes this change. Can you validate this is expected?